admin_guide/tcp_ingress_external_ports.adoc

```diff
@@ -12,6 +12,12 @@ toc::[]
 
 == Overview
 
+Note: This feature is only supported in non-cloud deployments.  For
+cloud (GCE/AWS/OpenStack) deployments, services of type
```

Comment:
Yes, OpenStack supports autoprovisioning of a loadbalancer for a service of type loadbalancer just like aws and gce.